streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsMetadataStateTest.java

```diff
@@ -88,7 +92,9 @@ public Object apply(final Object value) {
             }
         });
 
-        builder.globalTable("global-topic", "global-table");
+        builder.globalTable("global-topic",
+                            Consumed.with(null, null),
```

Comment:
This makes me thinking... should we have overloads for 1) topic, 2) topic+consumed, 3) topic+materialized and 4) topic+consumed+materialized ?

Reply:
EDIT: or maybe we can allow users to specify `null` only for 4)? In that case we can keep not adding 3).

Reply:
I think maybe we add the 4th overload